dist/index.html

```diff
@@ -31,11 +31,12 @@
         </div>
         <nav class="header-menu">
           <ul>
-            <li><a href="#">Research</a></li>
-            <li><a href="#">Data</a></li>
-            <li><a href="#">Blog</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Donate</a></li>
+            <li><a href="#">STATE</a></li>
```

Comment:
For these menu items, if we want them to be uppercase, I recommend [doing it with CSS styles](http://tympanus.net/codrops/css_reference/text-transform/)--otherwise people won't remember to type them in all caps on the backend.

A rule like `.header-menu > ul > li { text-transform: uppercase; }` *should* do it.